### PR TITLE
Process: Added missing tests etc.

### PR DESCRIPTION
**Pull request by woodyrew**
Added/Updated:

- Unit Tests
- Changelog
- README.
- Travis to include Node `v6`

_Note_: **Not** bumped minor version yet.

### CODE REVIEW

**Comment by woodyrew** [comment]
@Ajedi32 I'm happy to commit this but wanted a second pair of eyes. Also have a look at f368c7de please.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
-## [Unreleased]
+## [2.2.0] - August 11, 2016
+### Added
+* `process` to process files and plugins without writing files out.  ([#240])
```
Comment (Ajedi32) [inline CHANGELOG.md]: Why are there two "Added" sections now?

**Comment by Ajedi32** [comment]
Looks pretty good. A couple things should probably be added to the CHANGELOG before you release this though:

- I noticed earlier that 2fd4ae17882356289bd2db824a361538c0fd4347 is missing a CHANGELOG entry.
- The link references at the bottom of the CHANGELOG need to be updated to account for the new version

**Comment by woodyrew** [comment]
Made changes to changelog and made `.build()` DRY.
Bumped minor version

**Comment by Ajedi32** [comment]
:+1: LTGM

**Comment by woodyrew** [comment]
@Ajedi32 Do you have permission to publish to npm? I don't.

**Comment by Ajedi32** [comment]
No I don't. Looks like that's currently limited to @dominicbarnes and @ianstormtaylor

**Comment by dominicbarnes** [comment]
Do we need to make that list bigger? Or should I just publish v2.2.0?

**Comment by dominicbarnes** [comment]
@woodyrew @Ajedi32 let me know what your npm usernames are, I'll grant you both access.

**Comment by Ajedi32** [comment]
My username on npm is the same as here: https://www.npmjs.com/~ajedi32

**Comment by woodyrew** [comment]
I just noticed the tag `npm version` creates prefixes with a `v`, the other tagged versions don't have that prefix. Is it an issue? I think npm ignores the v anyway.
@dominicbarnes my npm username is `woodyrew`

**Comment by Ajedi32** [comment]
Conventionally version tags in git are prefixed with `v`. At least, that's how it's been on most other projects I've seen. Going forward, I'm fine with using that convention for new version tags.

Also, I just noticed the links to the commit diffs in the CHANGELOG are broken because I incorrectly assumed the version tags were prefixed with `v` when I created them. I wonder if it'd be worth it to go back and re-tag all the older releases. Looks like `git tag` does allow backdating tags to previous dates: https://git-scm.com/docs/git-tag#_date_formats (Though obviously whatever we decide to do, the old, unprefixed version tags should _not_ be removed.)

**Comment by woodyrew** [comment]
Yes, a git tag is essentially just a pointer to a commit. You can have many tags on one commit. I'm happy for them to be added. :+1:

**Comment by dominicbarnes** [comment]
The only thing that matters in `npm` is what you see in the `version` prop in the `package.json`. I generally don't use the `v` prefix, but many other projects do, so it's really a matter of preference. It would be nice if you continued the same pattern for this library as well, but no big deal if you decide not to.

I've added you both as npm owners. Publish away!

**Comment by woodyrew** [comment]
Published `2.2.0`. :tada:

I tend to use `npm version patch|minor|major` to bump npm versions - this adds `v{version number}` as the tag.
Personally, I prefer the `v` prefix but I agree the version tags should be consistent and I understand wanting to continue having version tags without the prefix.

**Pull request merged**